web/src/main/java/org/fao/geonet/monitor/onlineresource/CheckInfo.java

```diff
@@ -1,11 +1,29 @@
 package org.fao.geonet.monitor.onlineresource;
 
 public class CheckInfo {
-    public boolean status;
-    public long timestamp;
+
+    private CheckResult checkResult;
+    private long timestamp;
```

Comment:
OnlineResourceInfo tries to access this private variable
https://github.com/aodn/core-geonetwork/blob/840cc499723d79ec01767406f8e9760918410338/web/src/main/java/org/fao/geonet/monitor/onlineresource/OnlineResourceInfo.java#L43